src/components/TxSignSummary.vue

```diff
@@ -0,0 +1,137 @@
+<template>
+  <tx-box-details :assets="thisWalletGets" :type="'success'">
+    <p>Assets <strong>incoming</strong> to your wallet</p>
+  </tx-box-details>
+  <tx-box-details :assets="thisWalletLoses" :type="'danger'">
```

Comment:
I would like to keep `danger` as an exclusive token burning indicator, better to use `warning` here.